opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/SpanShim.java

```diff
@@ -47,6 +47,10 @@ public SpanShim(TelemetryInfo telemetryInfo, io.opentelemetry.api.trace.Span spa
     this(telemetryInfo, span, Baggage.empty());
   }
 
+  public SpanShim(TelemetryInfo telemetryInfo, io.opentelemetry.api.baggage.Baggage baggage) {
```

Comment:
The arguments don't match the other overloads. What do you think about having the caller or this call `new SpanShim(telemetryInfo, Span.getInvalid(), baggage)` instead?

Reply:
Done ;)